### ISSUE TITLE
Errors in setup

### ISSUE THREAD

**Issue by AbdulMalikDev** [PalisadoesFoundation/talawa#132]
Hi i'm a 3rd year Computer Science student from India.

I'm facing a few errors in setting up the dev environment, would be really grateful if someone could help me out here.

The first step i did was clone the project as mentioned

```
git clone https://github.com/PalisadoesFoundation/talawa.git
cd talawa
```

Got all packages

```
flutter pub get
```

Errors im facing are mostly about missing imports. Many files dont seem to be existing in the path that are mentioned in the imports. 

For example in the ``` talawa/lib/controllers/activity_controller.dart ``` file there is an import which does not exist :

```
import 'package:talawa/view_models/vm_add_activity.dart';
```

I've checked the ``` talawa/view_models ``` folder and there is no file named "**vm_add_activity.dart**"

Other such imports include : 

Location : ``` talawa/lib/controllers/activity_controller.dart ```
Error :
```
import 'package:talawa/model/activity.dart';
```

Location : ``` talawa/lib/controllers/note_controller.dart ```
Error :
```
import 'package:talawa/model/note.dart';
```

Formal Error : 

``` Target of URI doesn't exist: 'package:talawa/views/widgets/chat_message.dart'.
Try creating the file referenced by the URI, or Try using a URI for a file that does exist.darturi_does_not_exist ```

**Comment by AbdulMalikDev** [comment]
@SriramPatibanda Consider giving a like to this issue ( by pressing the smiley button on the top right ) to increase visibility of this issue.

**Comment by utkarshshendge** [comment]
@AbdulMalikDev maintainers are working on it and its expected to be fixed by 14th march.
see #111

**Comment by yasharth291** [comment]
#92  same

**Comment by Sagar2366** [comment]
@AbdulMalikDev try cloning latest master. Let us know if you still get same issue.

**Comment by DeltaHarbinger** [comment]
Hello @AbdulMalikDev, @SriramPatibanda, and @utkarshshendge. We merged #151 to master which seems to fix most problems. Check it out, we haven't heard of anyone having problems building since then
